devfiles/php-laravel/devfile.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: php-laravel
+projects:
+-
+  name: laravel-realworld-example-app
+  source:
+    type: git
+    location: "https://github.com/gothinkster/laravel-realworld-example-app.git"
+components:
+-
+  type: chePlugin
+  id: redhat/php/latest
+  memoryLimit: 1000Mi
+-
+  type: chePlugin
+  id: redhat/php-debugger/latest
+  memoryLimit: 256Mi
+-
+  type: dockerimage
+  alias: tools
+  image: eclipse/php:7.1-che7
+  env:
+    - name: HOME
+      value: /home/user
+    - name: PS1
+      value: $(echo ${0})\\$
+  memoryLimit: 512Mi
+  endpoints:
+  - name: '8080/tcp'
+    port: 8080
+  mountSources: true
+  volumes:
+    - name: composer
+      containerPath: "/home/user/.composer"
+-
+  type: dockerimage
+  alias: mysql
+  image: centos/mysql-57-centos7
+  env:
+    - name: MYSQL_USER
+      value: homestead
+    - name: MYSQL_PASSWORD
+      value: secret
+    - name: MYSQL_ROOT_PASSWORD
+      value: secret
+    - name: MYSQL_DATABASE
+      value: homestead
+  memoryLimit: 256Mi
+  endpoints:
+    - name: 'db'
+      port: 3306
+      attributes:
+        discoverable: "true"
+  mountSources: false
+commands:
+-
+  name: Start Artisan Web Server
+  actions:
+  - type: custom
```

Comment:
Have you tested it? I wonder why it worked, maybe some validation is missing on workspace creation, but type here must be `exec`

Reply:
I've tested it and it is working. But to have `exec` type for the commands is better solution, changed it